### ISSUE TITLE
pipx installation fails "aiohttp-json-rpc 0.11.2 has requirement aiohttp<3.5,>=3, but you'll have aiohttp 3.5.4 which is incompatible"

### ISSUE THREAD

**Issue by marcdexet-cnrs** [Lothiraldan/balto#13]
## Context

Installation on a fresh python 3.7-slim-stretch docker image (that the same with any python 3.7 installation I have done)

Usage of pipx installation fails with 

```
ERROR: aiohttp-json-rpc 0.11.2 has requirement aiohttp<3.5,>=3, but you'll have aiohttp 3.5.4 which is incompatible.
Could not find package balto. Is the name correct?
```

## Reproduction 

Use a virtualenv or this Dockerfile

```
from python:3.7-slim-stretch

RUN python3 -m pip install pipx
RUN pipx install balto

```

**Comment by marcdexet-cnrs** [comment]
There's no hope because of https://github.com/pengutronix/aiohttp-json-rpc/issues/38 :(

**Comment by aurelBZH** [comment]
hello,
Thanks a lot for the work on balto. I had seen your presentation at the pyconf last year and i would like to try balto. I have the same problem. Is there a plan for a patch ? 
thanks
Aurélien

**Comment by Lothiraldan** [comment]
I'm restarting working on Balto and I think I'm likely gonna drop the dependency all-together

**Comment by Lothiraldan** [comment]
I just released the version 1.0.5 of balto which replace aiohttp server by fastapi, enjoiy
